heron/tools/ui/src/python/main.py

```diff
@@ -614,9 +653,11 @@ def cli(
     host: str, port: int, base_url_option: str, tracker_url_option: str, verbose: bool
 ) -> None:
   """Start a web UI for heron which renders information from the tracker."""
-  global base_url, tracker_url
+  global base_url, tracker_url, Log
   base_url = base_url_option
-  log.configure(level=logging.DEBUG if verbose else logging.INFO)
+  log_level = logging.DEBUG if verbose else logging.INFO
+  log.configure(log_level)
+  Log = log.Log
```

Comment:
I assume the purpose of this line was to update the globally defined `Log`? If this is the case, we need the `Log` variable to be defined as `global` as it is now.

Reply:
I made my changes because the Debug logging didn't seem to work until I got a fresh copy of `log.Log` which is defined in `heron/common/src/python/utils/log.py`.  I can put these 3 lines back to the original line I replaced. Feel free to submit any edit suggestions. I've never really done any python coding, so not sure how best to handle the logging initialization.

Reply:
> I can put these 3 lines back to the original line I replaced. Feel free to submit any edit suggestions.

If you need a fresh copy of the logger here we will need to rename the variable or suppress the style check warnings. The issue that then crops up is that the `Log` variable is assigned in the method where it is declared, but it is never used. That is why I thought your intent is to update the globally defined `Log`.